.gitignore

```diff
@@ -8,3 +8,4 @@ build/
 packages
 .idea/
 
+pubspec.lock
```

Comment:
What is the advantage of having pubspec.lock checked in? I see only the disadvantage of not being able to reproduce a past dependency configuration, for example for a specific tag.